components/templates/NewsPage/NewsArticle.js

```diff
@@ -22,16 +22,13 @@ export default function NewsArticle({ data }) {
     subtitle,
     contacts,
     links,
-    /* eslint-disable camelcase */
-    more_information: moreInformation,
-    release_date: releaseDate,
-    /* eslint-enable camelcase */
+    moreInformation,
     headline,
     releaseUrl,
   } = data;
 
   const { t } = useTranslation();
-  const localizedDate = useDateString(date || releaseDate);
+  const localizedDate = useDateString(date);
```

Comment:
trying to have fewer release special properties and instead use the release fetch to parse the data into the existing news article shape